### PR TITLE
make: add vet target

### PR DESCRIPTION
**Pull request by Rolinh**

### CODE REVIEW

File: .github/workflows/unittests.yml [hunk 1]
```diff
@@ -23,4 +23,4 @@ jobs:
         export PATH=${PATH}:${GOPATH}/bin
         GO111MODULE=off go get -u github.com/gordonklaus/ineffassign
         GO111MODULE=off go get -u golang.org/x/lint/golint
-        make check-fmt lint test
+        make lint vet test
```
Comment (michi-covalent) [inline .github/workflows/unittests.yml]: no more ./contrib/scripts/check-fmt.sh?

Comment (Rolinh) [reply .github/workflows/unittests.yml]: It's a `lint` dep so it was effectively run twice. I think just once is enough :)

Comment (michi-covalent) [reply .github/workflows/unittests.yml]: maybe it'll catch something the second time :)

File: .github/workflows/unittests.yml [hunk 2]
```diff
@@ -23,4 +23,4 @@ jobs:
         export PATH=${PATH}:${GOPATH}/bin
         GO111MODULE=off go get -u github.com/gordonklaus/ineffassign
         GO111MODULE=off go get -u golang.org/x/lint/golint
-        make check-fmt lint test
```
Comment (glibsm) [inline .github/workflows/unittests.yml]: why was check-fmt removed?

Comment (Rolinh) [reply .github/workflows/unittests.yml]: See ab53f3f2b507eff8210e0edcf5bc784038b28563 or https://github.com/cilium/hubble/pull/178#discussion_r388561406

Comment (glibsm) [reply .github/workflows/unittests.yml]: Then why wasn't that file deleted? I wouldn't have any questions then :)

Comment (Rolinh) [reply .github/workflows/unittests.yml]: No file was deleted as part of this PR :thinking:

Comment (michi-covalent) [reply .github/workflows/unittests.yml]: seems like this got more confusing than necessary :) maybe we should have a single target that does all the formatting/linting/vet checks.

Comment (Rolinh) [reply .github/workflows/unittests.yml]: @michi-covalent See https://github.com/cilium/hubble/pull/181

**Pull request merged**